gitensite/apps/bookinfo/migrations/0005_auto_20180124_2104.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
could you squash these migrations into one?
to do this: 
1. migrate back to 0002
2. delete the 3 migrations (and the .pyc files!)
3. makemigrations
4. migrate